src/thunderbots/software/launch/ai_mrf.launch

```diff
@@ -0,0 +1,21 @@
+<launch>
+
+    <!-- Launch the network_input node -->
+    <node name="network_input" pkg="thunderbots" type="network_input" output="screen">
+    </node>
+
+    <!-- Launch the ai logic node -->
+    <node name="ai_logic" pkg="thunderbots" type="ai_logic" output="screen">
+    </node>
+
+    <!-- Launch the radio_communication node -->
+    <node name="radio_communication" pkg="thunderbots" type="radio_communication" output="screen">
+    </node>
+
+    <!-- Launch the dynamic parameters node -->
+    <node name="parameters" pkg="thunderbots" type="parameters" output="screen"/>
+
+    <!-- Launch the rqt_reconfigure gui -->
+    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>
+
+</launch>
```

Comment:
I've got a quick way to unify the commonalities between this and `ai_grsim.launch`, but I think it's easier if I just open a quick PR for that.